docs/projects/ad488x_fmc_evb/index.rst

```diff
@@ -0,0 +1,287 @@
+.. _ad488x_fmc_evb:
+
+AD488X-FMC-EVB HDL project
+================================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`EVAL-AD4880-FMC <EVAL-AD4880-FMC>` is designed to demonstrate the
+:adi:`AD4880 <AD4880>`
+performance.
+
+The :adi:`EVAL-AD4880-FMC <EVAL-AD4880-FMC>`  HDL design supports the following
+:adi:`AD4880 <AD4880>` features:
+
+* Single/Dual lane DDR data capture
+* Self synchronization using the fixed pattern and bit-slip feature
+* Capturing digital averaging filter with up to 2^10 decimation
+* Analog-to-digital converter (ADC) configuration via serial peripheral
+  interface (SPI)
+* Sampling rate capability between 1.25 MSPS and 40 MSPS
+* Synchronized capture between the channels
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD4880-FMC <EVAL-AD4880-FMC>`
+
+Supported devices
+-------------------------------------------------------------------------------
+-  :adi:`AD4880`
+-  :adi:`AD9508`
+-  :adi:`ADF4350`
+-  :adi:`LT6236`
+-  :adi:`ADP7182`
+-  :adi:`ADA4945-1`
```

Comment:
Same comment about the dash and space + you need an extra empty line after the section title

```suggestion

- :adi:`AD4880`
- :adi:`AD9508`
- :adi:`ADF4350`
- :adi:`LT6236`
- :adi:`ADP7182`
- :adi:`ADA4945-1`
```